crates/rust-analyzer/src/cli/analysis_stats.rs

```diff
@@ -361,3 +376,9 @@ fn shuffle<T>(rng: &mut Rand32, slice: &mut [T]) {
 fn percentage(n: u64, total: u64) -> u64 {
     (n * 100).checked_div(total).unwrap_or(100)
 }
+
+fn syntax_len(node: SyntaxNode) -> usize {
```

Comment:
Maybe print both to not underestimate the non-macro-generated code?

Reply:
I feel that ws should mostly be free, and there's a value in printing fewer numbers